automata/fa/nfa.py

```diff
@@ -211,7 +209,7 @@ def from_regex(
             The regex to construct an equivalent NFA for.
         input_symbols : Optional[AbstractSet[str]], default: None
             The set of input symbols to create the NFA over. If not
-            set, defaults to all ascii letters and digits.
+            set, defaults to all letters found in the regex.
```

Comment:
```suggestion
            set, defaults to all non-reserved characters found in the regex.
```
Now that we can have arbitrary unicode, we should talk about this in terms of characters and not letters.